src/dispatch/incident/flows.py

```diff
@@ -497,77 +497,21 @@ def incident_active_flow(incident_id: int, command: Optional[dict] = None, db_se
     # we load the incident instance
     incident = incident_service.get(db_session=db_session, incident_id=incident_id)
 
-    if incident.status == IncidentStatus.active:
-        if command:
-            convo_plugin = plugins.get(INCIDENT_PLUGIN_CONVERSATION_SLUG)
-            convo_plugin.send_ephemeral(
-                command["channel_id"],
-                command["user_id"],
-                "Incident Already Active Notification",
-                blocks=[
-                    {
-                        "type": "section",
-                        "text": {
-                            "type": "plain_text",
-                            "text": "The incident is already active. Aborting command...",
-                        },
-                    }
-                ],
-            )
-        return
-
-    # we update the status of the incident
-    update_incident_status(db_session=db_session, incident=incident, status=IncidentStatus.active)
-
-    log.debug(f"We have updated the status of the incident to {IncidentStatus.active}.")
-
     # we update the status of the external ticket
     update_incident_ticket(
         incident.ticket.resource_id,
         incident_type=incident.incident_type.name,
         status=IncidentStatus.active.lower(),
     )
 
-    log.debug(f"We have updated the status of the external ticket to {IncidentStatus.active}.")
-
-    # we update the conversation topic
-    set_conversation_topic(incident)
-
-    # we send the active notifications
-    send_incident_status_notifications(incident, db_session)
-
-    log.debug("We have sent the incident active notifications.")
-
```

Comment:
When are we calling these functions when we call this flow directly from the Slack code when we run `/dispatch-mark-active`?

Reply:
These are now handled by the `update` command.

Reply:
We are not calling the `update_flow` function when we run the command. Same for the stable and closed ones. See https://github.com/Netflix/dispatch/blob/develop/src/dispatch/plugins/dispatch_slack/views.py#L504

Reply:
We could add status to the update dialog and get rid of the mark [active|stable|closed] commands like we've discussed in the past.